resources/views/admin/news/index.blade.php

```diff
@@ -0,0 +1,58 @@
+@extends('admin.layouts.master')
+@section('content')
+<div class="row">
+@if(Session::has('success'))
+  <div class="alert alert-success">
+      {{ Session::get('success') }}
+  </div>
+@endif
+  <div class="col-xs-12">
+    <div class="box">
+      <div class="box-header">
+        <h3 class="box-title">{{trans('news.table')}}</h3>
+        <div class="box-tools">
+        </div>
+      </div>
+      <div class="box-body table-responsive no-padding">
+        <table class="table table-hover">
+          <tr>
+            <th>#</th>
+            <th style="width:15%">{{trans('news.title')}}</th>
+            <th style="width:8%">{{trans('categories.category')}}</th>
+            <th style="width:8%">{{trans('news.user')}}</th>
+            <th>{{trans('news.content')}}</th>
+            <th>{{trans('news.image')}}</th>
+            <th>{{trans('admin.action')}}</th>
+          </tr>
+          @foreach($news as $item)
+          <tr>
+            <td>{{$item->id}}</td>
+            <td>{{$item->title}}</td>
+            <td>{{$item->category->name}}</td>
```

Comment:
this code have n+1 problem. Pls use Laravel Eager loading
https://laravel.com/docs/5.4/eloquent-relationships#eager-loading